docs/UserGuide.md

```diff
@@ -6,13 +6,50 @@
 
 # CCA Manager User Guide
 
-CCA Manager is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+<!--
+- Clear and engaging introduction or welcome note that sets the tone for the UG.
+- What this User Guide aims to achieve
+- What this software is about
+    - Clear and engaging introduction or welcome note that sets the tone for the UG. 
+    - Identifies the target user or audience and makes appropriate assumptions, such as the level of relatedness, comprehension, and prior knowledge. 
+-->
+Welcome to CCA Manager's User Guide! CCA Manager is a **contact manager designed to simplify the management of CCAs and enhance your administrative efficiency**, regardless of whether you're overseeing a sports team, academic club, any other extracurricular activity, or simply a CCA participant. It has a minimal and intuitive GUI where most actions are performed via commands, making it a pleasure to use. If you can type fast, CCA Manager can get your admin tracking done faster than traditional GUI apps.
+
+In this user guide, we'll walk you through the essential steps to harness the full potential of CCA Manager. Whether you're a CCA Executive Committee Member or simply a CCA participant, our guide will provide you with the knowledge and tools you need to make the most of our app.
+ 
+This user guide does not assume any prior experience with administrative tools or command interfaces and is accessible and self-contained. If you have further questions while reading this document or while using our app, visit our [FAQ](#faq). If your question isn't answered there, feel free to visit our [project repository](https://github.com/AY2324S2-CS2103T-W11-2/tp/issues) and raise an issue.
```

Comment:
I guess it does assume some knowledge. They should at least be able to enter commands like in telegram, so we should state that.

Reply:
Sure